src/test/java/com/taskadapter/redmineapi/WikiManagerIT.java

```diff
@@ -1,20 +1,25 @@
 package com.taskadapter.redmineapi;
 
-import com.taskadapter.redmineapi.bean.Attachment;
-import com.taskadapter.redmineapi.bean.Project;
-import com.taskadapter.redmineapi.bean.User;
-import com.taskadapter.redmineapi.bean.WikiPage;
-import com.taskadapter.redmineapi.bean.WikiPageDetail;
+import com.taskadapter.redmineapi.bean.*;
```

Comment:
it is better to not use wildcard imports , they lead to class conflicts sooner or later. IDEs typically allow configuring number of imported classes before using wildcard imports in auto-reformat. if you set it to 500+, the IDE will never use wildcard imports.

Reply:
thank you for advice